src/renderer/appIde/project/project-node.ts

```diff
@@ -117,12 +113,8 @@ export function getNodeName (node: ProjectNode | string): string {
  * @returns Extension part of the project node
  */
 export function getNodeExtension (node: ProjectNode | string): string {
-  const filename = getNodeFile(node);
-  if (!filename) {
-    return "";
-  }
-  const fileParts = filename.split(".");
-  return fileParts.length > 0 ? "." + fileParts.slice(1).join(".") : "";
+  const fullPath = typeof node === "string" ? node : node.fullPath;
+  return fullPath ? path.extname(fullPath) : "";
```

Comment:
I dared to re-implement the family of node getters with path API, as we were getting '.git\\COMMIT_EDITMSG' as a file extension, from '%CURRENTUSER%\\Documents\\Projects\\SomeProject\\.git\\COMMIT_EDITMSG' path, which is not an expected result.